### PR TITLE
🎉 Add the papers release post

### PR DESCRIPTION
**Pull request by tomhoule**

### CODE REVIEW

File: _posts/2017-05-24-announcing-papers.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+practical_dev_link:
+---
+
+We are happy to announce the first public release of [Papers](https://github.com/store2be/papers), an HTTP service that receives LaTeX templates, variables and assets needed for the generated LaTex file, and generates good looking PDFs. It is designed to be stateless, safe, reliable, easy to use and to deploy. We even have a second binary to test your templates locally before deploying them.
+
+## What? Why?
+
+We needed a PDF generation service at store2be for a new feature in our advertising space manager. We investigated four options:
+
+- use a ruby library like [prawn](http://prawnpdf.org/),
```
Comment (peterfication) [inline _posts/2017-05-24-announcing-papers.md]: use a pure PDF Ruby library ...

File: _posts/2017-05-24-announcing-papers.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+- generate PDFs from HTML and CSS with something like [wkhtmltopdf](https://wkhtmltopdf.org/),
```
Comment (peterfication) [inline _posts/2017-05-24-announcing-papers.md]: I'd make the first chars here uppercase

File: _posts/2017-05-24-announcing-papers.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+- Have the Papers service running somewhere so you can make HTTP calls to it
+- Have the ([Tera](https://github.com/Keats/tera)) templates and assets hosted somewhere (we use S3/CloudFront)
+- Have an HTTP endpoint in your app where the PDF or errors will be posted
+
+Generating a PDF is then a simple matter of posting a JSON object describing your wish to the `/submit` endpoint of your Papers service. For example:
+
+```json
+{
+  "assets_urls": ["https://static.cook.example.com/spaghetti.jpg", "https://static.cook.example.com/pesto.jpg"],
```
Comment (peterfication) [inline _posts/2017-05-24-announcing-papers.md]: make some linebreaks here so it is easier to see that it is an array

File: _posts/2017-05-24-announcing-papers.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+categories:
+- Rust
+- Open source
+- Latex
+author_staff_member: 03_tom
+medium_link:
+practical_dev_link:
+---
+
+We are happy to announce the first public release of [Papers](https://github.com/store2be/papers), an HTTP service that receives LaTeX templates, variables and assets needed for the generated LaTex file, and generates good looking PDFs. It is designed to be stateless, safe, reliable, easy to use and to deploy. We even have a second binary to test your templates locally before deploying them.
```
Comment (peterfication) [inline _posts/2017-05-24-announcing-papers.md]: [first public release](https://github.com/store2be/pape-rs/releases/tag/0.1.0)

File: _posts/2017-05-24-announcing-papers.md (hunk 2 above)
Comment (brainlessdeveloper) [inline _posts/2017-05-24-announcing-papers.md]: :+1:

**Comment by brainlessdeveloper** [comment]
:100: :fire:

**Pull request merged**